persistence-jdbc/javadsl/src/multi-jvm/resources/reference.conf

```diff
@@ -1 +1,15 @@
 jdbc-defaults.slick.profile = "slick.jdbc.H2Profile$"
+
+
+
+# Aggressively restart and ensure the cluster-distributed, readside actors so tests don't take that long.
```

Comment:
do we still need this? Because since the `EnsureActorActive` has 30 sec timer, this one is actually useless

Reply:
We don't need this actually but I'd rather get rid of `ensure-active-interval` than get rid of `failure-exponential-backoff`. When we implement #999, if we don't have `failure-exponential-backoff` tuned for this tests, the tests will break again.